### PR TITLE
Increase connection timeout for imports

### PR DESCRIPTION
**Pull request by markus1189**
### TL;DR

Set the response timeout from 1s to 30s, because 1s is much too optimistic.

### Why

From time to time I have the _problem_ luck of being with a rather modest internet connection (~376Kb/s) and I noticed that even running something simple like:

```
let not = https://ipfs.io/ipfs/QmcTbCdS21pCxXysTzEiucDuwwLWbLUWNSKwkJVfwpy2zK/Prelude/Bool/not
in  not True
```

via `dhall` fails with `ConnectionTimeout`.  I saw that the connection response timeout is set to `1 second`, which is much too short for a bad internet connection. 

### How
To address this, I raised the time limit from `1 second` to `30 seconds`, because that is also the time that is used in the default `managerResponseTimeout` (https://hackage.haskell.org/package/http-client-0.5.6.1/docs/src/Network-HTTP-Client-Types.html#ManagerSettings).

With that setting I can finally use `dhall` even when I don't have a fast internet connection available. :D

### CODE REVIEW

**Comment by Gabriel439** [comment]
Yeah, great idea!  Sorry about picking such a strict default initially
